.github/workflows/set-date.yml

```
@@ -1,17 +1,18 @@
-name: Set the End Date in the project
+name: Set the Date in the project
 
 on:
   issues:
-    types: [closed]
+    types: [assigned, closed]
 jobs:
   set_date:
     runs-on: ubuntu-22.04
     steps:
-      - name: Get project data
+      - name: Get project data for start date
         env:
           GITHUB_TOKEN: ${{ secrets.KEPTN_BOT_PROJECT_TOKEN }}
           ORGANIZATION: keptn
           PROJECT_NUMBER: 18
+        if: ${{ github.event.action == 'assigned' }}
```

you don't need the curly braces in `if` blocks
```suggestion
        if: github.event.action == 'assigned'
```